tests/linked_list_test.c

```diff
@@ -67,11 +67,7 @@ static int test_linked_list_order_push_back_pop_front() {
     ASSERT_INT_EQUALS(list_size - 4, aws_linked_list_length(&list), "List size should be %d.", (int)list_size - 4);
 
     aws_linked_list_clean_up(&list);
-    RETURN_SUCCESS("%s() pass", "test_linked_list_order_push_back_pop_front");
-
-failure:
-    aws_linked_list_clean_up(&list);
-    return FAILURE;
+    return SUCCESS;
```

Comment:
these should either just be return 0 or return AWS_OP_SUCCESS. The success macro is old, and I probably just need to delete it.